### DISCUSSION TITLE
Is this library applicable with dynamic feature modules

### DISCUSSION THREAD

**Discussion by chetanurmaliya** [bumble-tech/appyx#58] [Q&A]
Working on Jetpack Compose Navigation with multi module application using Kotlin DSL.
We have dynamic feature modules DFM1, DFM2, DFM3.

How to setup navigation graph and navigate between dynamic feature modules?

**Comment by LachlanMcKee** [comment]
Hi @chetanurmaliya, sorry for the delay getting back to you.

You can indeed use Appyx with dynamic feature modules. I am currently working on a sample that would show how this can be done.

However there are quite a few ways to solve this problem. What approach/libraries do you currently use to solve this? Perhaps even if my sample project doesn't work the same way, I could give you advice, or your suggestion could improve the sample.
